### PR TITLE
Removed non-working image host AllYouCanUpload

### PR DESCRIPTION
**Pull request by UtiluMark**
Part of #188

### CODE REVIEW

**Comment by xabolcs** [comment]
@UtiluMark, how does this relate to #49?

**Comment by UtiluMark** [comment]
@xabolcs In this PR the non-working image host AllYouCanUpload is removed, #49 is about the other image host ImageShack.

File: extension/chrome/content/screenshot/providers.js [hunk 1]
```diff
@@ -47,37 +47,13 @@ var ImageShack = {
 
   getReferer: function()
   {
-    return "http://www.imageshack.us/";
+    return "http://imageshack.us/";
```
Comment (xabolcs) [inline extension/chrome/content/screenshot/providers.js]: This change is unrelated.

File: extension/chrome/content/screenshot/providers.js [hunk 2]
```diff
@@ -47,37 +47,13 @@ var ImageShack = {
...
   },
 
   getSubmissionURL: function()
   {
-    return "http://www.imageshack.us/";
```
Comment (xabolcs) [inline extension/chrome/content/screenshot/providers.js]: This change is unrelated also.

**Comment by xabolcs** [comment]
Thanks @UtiluMark!

Didn't tested yet.

I don't like the unrelated changes, but for now I could accept them knowing that it would be overwritten soon by fixing #49.

File: extension/chrome/content/screenshot/providers.js (hunk 2 above)
Comment (UtiluMark) [inline extension/chrome/content/screenshot/providers.js]: The diff shows like I removed ImageShack and replaced AllYouCanUpload by ImageShack, while I only removed AllYouCanUpload and the www. before ImageShack.

Comment (xabolcs) [inline extension/chrome/content/screenshot/providers.js]: Removing the `www.` prefix from ImageShack's URL is not needed by AllYouCanUpload image host removal. :)
But I could accept for now, as I wrote before.

Fixing image host ImageShack is #49's business.

**Comment by UtiluMark** [comment]
@whimboo Please review and merge.

**Comment by xabolcs** [comment]
r+

**Comment by whimboo** [comment]
I would tend to say lets revert the www prefix removal. It's better to have listed the specific subdomain in case something will change on that host (which indeed may be very unlikely but lets be safe).

**Comment by UtiluMark** [comment]
@whimboo I removed the www prefix, because https://imageshack.us/ works, while https://www.imageshack.us/ produces an ssl_error_bad_cert_domain error.
I know we currently use http and not https, but as soon as that changes, the www prefix will need to be removed anyway.

**Comment by whimboo** [comment]
Interesting. The cert is only valid for `*.imageshack.com` but not `us`. Looks like its a broken SSL configuration on their end. So lets get this in for now but in the future lets ensure to have a separate PR for that.

**Comment by whimboo** [comment]
Merged the PR as https://github.com/mozilla/nightlytt/commit/497220120678f574d317a4ac576c5c7fd4b46bb1. Thanks!
